libs/MeshKernelApi/include/MeshKernelApi/GeometryList.hpp

```diff
@@ -33,10 +33,10 @@ namespace meshkernelapi
     struct GeometryList
     {
         /// @brief The value used as separator in coordinates_x, coordinates_y and values
-        double geometry_separator;
+        double geometry_separator = -999.0;
```

Comment:
Is this the same as constants::missing::doubleValue?

Reply:
yes, let's default it to the separator used internally. the client can always get the correct value using the API available in meshkernel.